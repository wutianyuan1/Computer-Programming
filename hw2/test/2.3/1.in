4
4
12
2
2
12
5
13
13
4
14
7
12
9
6
14
2
5
9
3
-1 	
8
14
8
5
6
-1
