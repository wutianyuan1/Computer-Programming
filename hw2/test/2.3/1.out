Please input the set A:
(resize) from 5 to 10
(resize) from 10 to 20
(resize) from 20 to 40
Please input the set B:
(resize) from 5 to 10
---Result---
6 14 2 5 9
